### PR TITLE
Add and start to refine layout styles

### PR DESCRIPTION
**Pull request by kylefiedler**
screen shot 2014-11-13 at 9 54 56 am

### CODE REVIEW

File: _sass/base/_typography.scss [hunk 1]
```diff
@@ -1,10 +1,21 @@
 body {
   -webkit-font-smoothing: antialiased;
-  background-color: $base-background-color;
   color: $base-font-color;
   font-family: $base-font-family;
   font-size: $base-font-size;
   line-height: $base-line-height;
+
+  &:after {
```
Comment (creuter) [inline _sass/base/_typography.scss]: Why does this need to be in a psuedo?

File: _sass/base/extends/_content-header.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+%content-header {
+  ont-size: em(14);
```
Comment (creuter) [inline _sass/base/extends/_content-header.scss]: `ont-size`

File: _sass/layout/_footer.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+footer {
+  background-color: $secondary-background-color;
+  color: white;
+  padding: 4em;
```
Comment (creuter) [inline _sass/layout/_footer.scss]: :abc:

File: _sass/layout/_hero.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
+.hero {
+  padding-bottom: 36vh;
```
Comment (creuter) [inline _sass/layout/_hero.scss:2]: `vh` is buggy with iOS7, so use your discretion. Here's a workaround: https://gist.github.com/pburtchaell/e702f441ba9b3f76f587 if you're up for it

File: _sass/layout/_hero.scss [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    $hero-line-padding: 0.5em;
+    color: white;
+    font-family: $base-font-family;
+    font-size: em(28);
+    font-weight: normal;
+    margin-left: 0.5em;
+
+    span {
+      $hero-header-background: $base-accent-color;
+      @include prefixer(box-decoration-break, clone, webkit moz o spec);
```
Comment (creuter) [inline _sass/layout/_hero.scss:15]: whoa

**Comment by creuter** [comment]
:ship:

File: _sass/layout/_hero.scss (hunk 4 above)
Comment (kylefiedler) [inline _sass/layout/_hero.scss:2]: Yea I figured once I got to smaller screens I'd deal with that then.

**Pull request merged**